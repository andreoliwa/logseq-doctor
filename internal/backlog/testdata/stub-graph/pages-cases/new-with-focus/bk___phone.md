- ((67c48e28-84c8-4061-9451-42beddd5a2df))
- Focus
- ((67cbf35b-332d-4fe7-89b0-767d54f5daae))
