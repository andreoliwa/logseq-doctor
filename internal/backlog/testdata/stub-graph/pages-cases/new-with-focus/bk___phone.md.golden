- ((67c48e28-84c8-4061-9451-42beddd5a2df))
- Focus
- New tasks [[quick capture]]
	- ((67cfe045-8ddb-48ba-a3d0-ccc5241c63a7))
- ((67cbf35b-332d-4fe7-89b0-767d54f5daae))
